### PR TITLE
Feature/related devices

### PR DESCRIPTION
**Pull request by hassan-yonomi**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/utils/messageProcessor.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class MessageProcessor {
     }
 
     if (_topic.type === Constants.Topic.REGISTRATION) {
-      this._client.deviceId = json && json.result ? json.result.deviceId : null;
+      // this._client.deviceId = json && json.result ? json.result.deviceId : null;
```
Comment (stewartjarod) [inline src/utils/messageProcessor.js]: rm

Comment (mikelittman) [reply src/utils/messageProcessor.js]: ⓨⓔⓢ

**Review by stewartjarod — APPROVED** [review]
I like this

File: src/utils/messageProcessor.js (hunk 1 above)
Comment (stewartjarod) [reply src/utils/messageProcessor.js]: 🥇

**Review by mikelittman — APPROVED** [review]
❗️💩👍

**Pull request merged**